src/frontend.ts

```diff
@@ -256,6 +268,31 @@ export class Frontend {
     }
   }
 
+  private translateInt(node: source.node.Int) : ReadonlyArray<IWrap<frontend.node.Node>> {
+    const { name, field, bytes, signed, littleEndian } = node;
+
+    let result = [
+      new this.implementation.node.Int(
+        new frontend.node.Int(this.id.id(name), field, bytes, signed, littleEndian, 0)
+      )
+    ];
+
+    // Break loops
+    this.map.set(node, result[0]);
+
+    let next = result[0];
+    for (let offset = 1; offset < node.bytes; offset++) {
+      result[offset] = new this.implementation.node.Int(
+        new frontend.node.Int(this.id.id(name + '_byte' + (offset + 1)), field, bytes, signed, littleEndian, offset)
```

Comment:
Nitpick: similar recommendation here. Perhaps even move `this.id.id()` to a separate statement too. Also, would be great opportunity to use template syntax here: `${name}_byte${offset + 1}`.